keyboards/handwired/3dp660/readme.md

```diff
@@ -0,0 +1,26 @@
+# 3dp660
+
+![](https://i.imgur.com/KWyU57el.jpg)
+
+### Teensy 2.0 Pinout
+| rows        | 0   | 1   | 2   | 3   | 4   |
+| ----------- | --- | --- | --- | --- | --- |
+| Teensy pin  | B0  | B1  | B2  | B3  | B7  |
+| qmk pin     | B0  | B1  | B2  | B3  | B7  |
+
+| columns     | 0   | 1   | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   | 10  | 11  | 12  | 13  | 14  |
+| ----------- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
+| Teensy pin  | D2  | D3  | C6  | C7  | D5  | D4  | D7  | B4  | B5  | B6  | F7  | F6  | F5  | F4  | F1  |
+| qmk pin     | D2  | D3  | C6  | C7  | D5  | D4  | D7  | B4  | B5  | B6  | F7  | F6  | F5  | F4  | F1  |
```

Comment:
You could do:

```suggestion
| rows            | 0   | 1   | 2   | 3   | 4   |
| --------------- | --- | --- | --- | --- | --- |
| Teensy/QMK pin  | B0  | B1  | B2  | B3  | B7  |

| columns         | 0   | 1   | 2   | 3   | 4   | 5   | 6   | 7   | 8   | 9   | 10  | 11  | 12  | 13  | 14  |
| --------------- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- | --- |
| Teensy/QMK pin  | D2  | D3  | C6  | C7  | D5  | D4  | D7  | B4  | B5  | B6  | F7  | F6  | F5  | F4  | F1  |
```

since the pin names are the same between Teensy and QMK. But I consider this optional.